src/renderer40k.ts

```diff
@@ -193,6 +196,34 @@ export class Renderer40k implements Renderer {
                     detachmentSheets.classList.remove('d-none')
                 }
             });
+        // ability to hide divs (abilities, rules, ...) from printing
+        optionsDiv.appendChild(document.createElement('span')).innerHTML = '&nbsp;&nbsp;&nbsp;&nbsp;';
+        this.renderCheckboxOption(optionsDiv, 'hideElements', 'Hide Elements from Printing',
+            (e: Event) => {
+                if ((e.target as any).checked) {
+                    const customize = document.createElement('style');
+                    customize.id = 'customize'
+                    customize.textContent =
+                        `.hide_able:hover {
```

Comment:
Add `cursor: pointer` to change the mouse to a pointer, which shows that the element is clickable